products/sle/main.pm

```diff
@@ -794,6 +794,12 @@ elsif (get_var("VIRT_AUTOTEST")) {
             loadtest "virt_autotest/virsh_external_snapshot";
         }
     }
+    elsif (get_var("ENABLE_SRIOV_NETWORK_CARD_PCI_PASSSHTROUGH")) {
+        loadtest "virt_autotest/restore_guests";
+        loadtest "virt_autotest/set_config_as_glue";
+        loadtest "virt_autotest/setup_dns_service";
+        loadtest "virt_autotest/sriov_network_card_pci_passthrough";
```

Comment:
This flow will definitely work well in our automation, what about for maintenance? @pdostal @grisu48 I guess you also have interest on enabling SR-IOV test for maintenance. Feel free to give your review!